app/controllers/clients_controller.rb

```diff
@@ -97,6 +97,8 @@ def update
     @transfer_users = current_user.department.eligible_users.where.not(id: current_user.id).pluck(:full_name, :id)
 
     @client.assign_attributes(client_params)
+
+    @client.valid?
```

Comment:
this line of code is unnecessary since we're not querying `@client.errors` before running `if @client.save` which will run validations and set `errors` if there are any.

Reply:
... I think